source/process/sg_general-guidelines.rst

```diff
@@ -93,3 +93,27 @@ Avoid
   The community manager posts questions and answers that they think are well-written.
 Do not use
   The community manager posts questions and answers that he or she thinks are well-written.
+
+Pull Request Review Best Practices
+--------------------------------------
+
+**Treat everyone with respect**
+
+ - Choose positivity over negativity at all times.
+ - Approach the review the same way regardless of whether the pull request came from a first-time contributor, or a long-time staff member.
+ - If there are multiple instances of an error/issue, summarize expectations in a general comment vs pointing out the same error/nuance multiple times.
+ - Be thankful - submitters really respond well to being praised and thanked for their work.
+
+**Seek first to understand and look for opportunities to help others learn**
+
+ - Don’t be afraid to ask for changes or clarify expectations until all concerns are addressed but keep a welcoming/inclusive tone.
+ - Do your best to link documentation as answers.
+
+**Respond promptly to requested reviews**
+
+ - Assume the requested review is urgent and blocking unless explicitly stated otherwise.
```

Comment:
@jasonblais I believe this can generate pressure on the reviewer, thoughts?

Reply:
@enahum Good feedback. What is the expectation from developers when a review is requested?

Reply:
I'd like to add that I copied the wording from this page: https://developers.mattermost.com/contribute/getting-started/code-review/ under the Core Committers section. Would a change affect the wording across the board and be a wider conversation) or adapted for this list only?

Reply:
I'm 0/5 if we change it or not.. just pointed out the fact, not sure what is the best way of addressing this. We can leave it as is and then circle back if needed

Reply:
Thanks Elias - I'm 0/5 as well, and having different sets of expectations might work better in different contexts. If there is an issue down the line, we'll sort it out. :)

Reply:
Since it's part of the dev code review process, propose we keep it and we can iterate if we hear feedback

@enahum @justinegeffen